src/views/RuleEngine/Bridge/BridgeDetail.vue

```diff
@@ -184,6 +187,14 @@ const isWebhook = computed(() => {
   return route.name === 'webhook-detail-stats'
 })
 
+const bridgeName = computed(() => {
+  const { name } = bridgeInfo.value
+  if (name && isWebhook.value) {
+    return name.replace(WEBHOOK_SUFFIX, '')
```

Comment:
can we use https://github.com/emqx/emqx-dashboard5/blob/01ea27d5f4daba445592276d2842a3b434a17515/src/hooks/Webhook/useWebhookUtils.ts#L52